### ISSUE TITLE
New Source Plugin won't retrieve Experience Bundle

### ISSUE THREAD

**Issue by jefersonchaves** [forcedotcom/cli#1061]
### Summary

The new plugin [source](https://github.com/salesforcecli/plugin-source) does not retrieve [Experience Bundles](https://developer.salesforce.com/docs/atlas.en-us.api_meta.meta/api_meta/meta_experiencebundle.htm).

### Steps To Reproduce:

1.  `sfdx force:source:retrieve -m ExperienceBundle:Community`.

### Expected result

Experience Bundle Metadata retrieved.

### Actual result

The start and end of the retrieval message are displayed, but no component is downloaded or displayed.

### Additional information

**SFDX CLI Version**: sfdx-cli/7.94.3-a4e7c7955b darwin-x64 node-v14.15.4

**SFDX plugin Version**

@salesforce/lwc-dev-server 2.10.0
├─ @oclif/plugin-help 2.2.3
└─ @oclif/plugin-update 1.3.10
@salesforce/sfdx-diff 0.0.6
functions 0.2.8
└─ @oclif/plugin-not-found 1.2.4
source 0.1.20

**OS and version**: Mac OS 11.4

**Comment by uip-robot-zz** [comment]
This issue has been linked to a new work item: [W-9512293](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07AH000000PTteYAG/view)

**Comment by shetzel** [comment]
Hi @jefersonchaves - thanks for trying the new source plugin!  I'm trying to reproduce this using the [ebikes repo](https://github.com/trailheadapps/ebikes-lwc) but it's working as I'd expect.  Here is the command I'm using:

`sfdx force:source:retrieve -m ExperienceBundle:E_Bikes1`

Here is my cli version: `sfdx-cli/7.107.0 darwin-x64 node-v14.15.1`

My Plugins:
```
@oclif/plugin-autocomplete 0.3.0 (core)
@oclif/plugin-commands 1.3.0 (core)
@oclif/plugin-help 3.2.2 (core)
@oclif/plugin-not-found 1.2.4 (core)
@oclif/plugin-plugins 1.10.0 (core)
@oclif/plugin-update 1.4.0-3 (core)
@oclif/plugin-warn-if-update-available 1.7.0 (core)
@oclif/plugin-which 1.0.3 (core)
@salesforce/sfdx-plugin-lwc-test 0.1.7 (core)
@salesforce/sfdx-trust 3.6.0 (core)
alias 1.1.10 (core)
apex 0.2.2 (core)
auth 1.6.1 (core)
config 1.2.8 (core)
custom-metadata 1.0.12 (core)
data 0.4.12 (core)
generator 1.1.8 (core)
limits 1.2.1 (core)
org 1.6.7 (core)
salesforce-alm 52.0.2 (core)
schema 1.0.7 (core)
sfdx-cli 7.107.0 (core)
source 0.1.20
telemetry 1.2.2 (core)
templates 52.0.0 (core)
user 1.3.0 (core)
```

Is the source plugin reported with: `sfdx which force:source:retrieve`?
Maybe your CLI version needs updating?
Maybe there is another plugin causing problems?  E.g., temporarily uninstall the plugins, try again, then reinstall?

**Comment by jefersonchaves** [comment]
Hey @shetzel,

Yes, now after upgrading the CLI (from 7.94.3-a4e7c7955b to 7.107.0-621a7d2 - I'm using the installer not npm), then it worked as expected.

See the plugins:
```
@oclif/plugin-autocomplete 0.3.0 (core)
@oclif/plugin-commands 1.3.0 (core)
@oclif/plugin-plugins 1.10.0 (core)
@oclif/plugin-warn-if-update-available 1.7.0 (core)
@oclif/plugin-which 1.0.3 (core)
@salesforce/lwc-dev-server 2.10.0
├─ @oclif/plugin-help 2.2.3
└─ @oclif/plugin-update 1.3.10
@salesforce/sfdx-diff 0.0.6
@salesforce/sfdx-plugin-lwc-test 0.1.7 (core)
@salesforce/sfdx-trust 3.6.0 (core)
alias 1.1.10 (core)
apex 0.2.2 (core)
auth 1.6.1 (core)
config 1.2.8 (core)
custom-metadata 1.0.12 (core)
data 0.4.12 (core)
functions 0.2.13
└─ @oclif/plugin-not-found 1.2.4
generator 1.1.7 (core)
limits 1.2.1 (core)
org 1.6.7 (core)
salesforce-alm 52.0.2 (core)
schema 1.0.7 (core)
sfdx-cli 7.107.0 (core)
source 0.1.21
telemetry 1.2.2 (core)
templates 52.0.0 (core)
user 1.3.0 (core)
```

**Comment by WillieRuemmele** [comment]
Glad upgrading the CLI worked 👍 . Closing the issue
